src/locales/fr/messages.json

```
@@ -40,9 +40,8 @@
   "INVALID_FOLDER_PATH_FOR_DELETE": "Le chemin du dossier soumis n'est pas valide pour la suppression",
   "INVALID_DOWNLOAD_PARAMETERS": "Un ou plusieurs paramètres de téléchargement ne sont pas valides",
   "LOCAL_FILE_NOT_FOUND": "Le fichier local n'est pas trouvé",
-  "S3_FILE_NOT_FOUND": "Le fichier S3 n'est pas trouvé",
   "UPLOAD_FILE_UNEXPECTED_ERROR": "Une erreur inattendue s'est produite lors du téléchargement de fichiers",
-  "DOWNLOAD_FILE_UNEXPECTED_ERROR": "Une erreur inattendue s'est produite lors du téléchargement d'un fichier",
+  "DOWNLOAD_FILE_UNEXPECTED_ERROR": "Le fichier ne peut pas être téléchargé. Veuillez réessayer plus tard.",
```

Maybe something like this ?

```suggestion
  "DOWNLOAD_FILE_UNEXPECTED_ERROR": "Le fichier n'a pas pu être téléchargé. Veuillez réessayer plus tard.",
```

---

Should we add "Si le problème persiste contactez support@graasp.org"